doc/changelog/03-notations/11841-master+distinguishing-ident-name-in-grammar-entries.rst

```diff
@@ -0,0 +1,13 @@
+- **Changed:**
+  In notations (except in custom entries), the misleading :n:`@syntax_modifier`
+  :n:`@ident ident` (which accepted either an identifier or
+  a :g:`_`) is deprecated and should be replaced by :n:`@ident name`. If
+  the intent was really to only parse identifiers, this will
+  eventually become possible, but only as of Coq 8.14.
```

Comment:
Here you say that it will become possible as of Coq 8.14 but in your updated plan this was rather Coq 8.15. Personally I'm OK with skipping the intermediate fatal warning step, but to be on the safe side, don't you want to say "in a future version" instead of making predictions?